### ISSUE TITLE
Missing CFN Resource Specifications from AWS Documentation

### ISSUE THREAD

**Issue by ScriptAutomate** [ScriptAutomate/aws-cfn-resource-specs#1]
> _**NOTE:** Some CFN Resource Specifications may be missing because they have yet to be officially published. As a result, I am including the GitHub issue links for reference on conversation around why they aren't included on the official documentation page._

The following CFN resource specifications are missing from the [AWS CloudFormation Resource Specification](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-resource-specification.html) landing page:

| Region  | Location  | GitHub Issue |
|---|---|---|
|  `ap-east-1` | [Asia Pacific (Hong Kong)](https://aws.amazon.com/blogs/aws/now-open-aws-asia-pacific-hong-kong-region/)  | [No Resource Specification for ap-east-1](https://github.com/awsdocs/aws-cloudformation-user-guide/issues/320) |
| `cn-north-1`  | [China (Beijing)](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-sample-templates-cn-north-1.html)  | [Missing CFN Resource Spec File Links for China](https://github.com/awsdocs/aws-cloudformation-user-guide/issues/425) |
| `cn-northwest-1`  | [China (Ningxia)](https://aws.amazon.com/blogs/aws/now-open-aws-china-ningxia-region/)  | [Missing CFN Resource Spec File Links for China](https://github.com/awsdocs/aws-cloudformation-user-guide/issues/425) |
| `us-gov-east-1` | [AWS GovCloud (US-East)](https://aws.amazon.com/blogs/aws/aws-govcloud-us-east-now-open/)  | [Missing CFN Resource Spec File Links for GovCloud](https://github.com/awsdocs/aws-cloudformation-user-guide/issues/426) |
| `us-gov-west-1`  | [AWS GovCloud (US-West)](https://aws.amazon.com/govcloud-us/)  | [Missing CFN Resource Spec File Links for GovCloud](https://github.com/awsdocs/aws-cloudformation-user-guide/issues/426) |
| `me-south-1`  | [Middle East (Bahrain)](https://aws.amazon.com/about-aws/whats-new/2019/07/announcing-the-new-aws-middle-east--bahrain--region-/)  | [Missing me-south-1 Specs](https://github.com/awsdocs/aws-cloudformation-user-guide/issues/421) |new/2019/07/announcing-the-new-aws-middle-east--bahrain--region-/)  | [Missing me-south-1 Specs](https://github.com/awsdocs/aws-cloudformation-user-guide/issues/421) | [ ] |

Their links can be found in the `cfn-python-lint` source code, maintained by the AWS CloudFormation team, here:

* [aws-cloudformation/cfn-python-lint/src/cfnlint/maintenance.py](https://github.com/aws-cloudformation/cfn-python-lint/blob/4767ceefd5091a48fe2cb75f5954820aa150aa6b/src/cfnlint/maintenance.py#L31-L53)

**Comment by ScriptAutomate** [comment]
All resources above now exist on the CFN resource spec landing page.

**Comment by ScriptAutomate** [comment]
I closed this too early. The documentation / links have been updated on the official website, but they have not been updated in this repository. I will keep this open until the repository reflects the website.

It looks like the website had been updated ~20 - 22 days ago, but that this repository never was.

File not updated since **March**: [aws-cloudformation-user-guide/doc_source/cfn-resource-specification.md](https://github.com/awsdocs/aws-cloudformation-user-guide/blob/master/doc_source/cfn-resource-specification.md)
